cms/core/blocks/glossary_terms.py

```diff
@@ -0,0 +1,43 @@
+from collections.abc import MutableSequence
+from typing import TYPE_CHECKING, Any
+
+from wagtail.blocks import ListBlock
+from wagtail.snippets.blocks import SnippetChooserBlock
+
+from cms.core.models import GlossaryTerm
+
+if TYPE_CHECKING:
+    from wagtail.blocks import StreamValue, StructValue
+
+
+class GlossaryTermsBlock(ListBlock):
+    def __init__(self, search_index: bool = True, **kwargs: Any) -> None:
+        super().__init__(SnippetChooserBlock(GlossaryTerm), search_index=search_index, **kwargs)
+
+    class Meta:
+        template = "templates/components/streamfield/glossary_section_block.html"
+
+    def get_context(self, value: "StreamValue", parent_context: dict | None = None) -> dict:
+        """Inject formatted glossary terms to be used with ONS Accordion component."""
+        context: dict = super().get_context(value, parent_context)
+        context["formatted_glossary_terms"] = [
+            {
+                "title": glossary_term.name,
+                "content": glossary_term.definition,
```

Comment:
```suggestion
                "content": richtext(glossary_term.definition),
```

with `richtext` imported as `from wagtail.templatetags.wagtailcore_tags import richtext`